ground/gcs/src/plugins/uavobjectbrowser/treeitem.cpp

```diff
@@ -45,7 +45,7 @@ HighLightManager::HighLightManager(long checkingInterval, QTime *currentTime)
     m_expirationTimer.start(checkingInterval);
     connect(&m_expirationTimer, &QTimer::timeout, this, &HighLightManager::checkItemsExpired);
 
-    if (currentTime == NULL)
+    if (m_currentTime == NULL)
         m_currentTime = new QTime;
```

Comment:
Should probably be QTime::currentTime()? QTime default ctor returns a null time. But uhhh.... the check looked correct before?